Wrappers.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "Cat.h"
+#include "Fruit.h"
+#include "Person.h"
+#include "TrashBin.h"
+#include "Treadmill.h"
```

Comment:
Don't include what you don't use.  
All of these types can be forward declared here.  include these files in the CPP file.
- [ ] revise